.github/workflows/deployment-test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           go-version: 1.17
       - uses: ./.github/actions/restore-node
         with:
-          node-version: 16.x
+          node-version: 18.x
```

Comment:
I'll force integration to test if 18 works for the deployment test, but we may want to not poke the bear and stick with 16 here for now.

Reply:
Yeah looks like the version of node 18 available on GH actions is incompatible with the ancient version of ubuntu we're still using for the deployment test. I think we have an issue somewhere about upgrading the ubuntu version, or at least GH action is getting insistent, so maybe we can take care of switching to node 18 then.

Reply:
Ack.